packages/app/src/settings/SettingsSection.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <section>
-    <h2 class="font-medium text-[16px] text-gray-900 leading-[24px] inline-flex items-baseline align-middle">
+    <h2
+      :id="anchor"
```

Comment:
Moving the "id" to the `section` element instead of the `h2` gives a bit more padding at the top when it scrolls, but totally an optional change.

Reply:
Made this change in https://github.com/cypress-io/cypress/pull/27096/commits/3c3ca4d032f01bc93cee09eeb60507e06abbe11c